Sources/AblyChat/Messages.swift

```diff
@@ -1,10 +1,35 @@
 import Ably
 
+public protocol MessageSubscriptionResponse: Sendable {
+    func unsubscribe() async -> Void
```

Comment:
Why the `unsubscribe`? I intentionally chose to omit this in 20e7f5f652142c313a49767ef98cc459c0fc82e2; do you disagree with the rationale described there? If so, please add an explanation to commit message.

Reply:
solved by using `MessageSubscription` again